### ISSUE TITLE
Understanding the output

### ISSUE THREAD

**Issue by dhgoratela** [ProsusAI/finBERT#61]
I need help in understanding the output of the model. My questiions are as follows:

1. The **model output** is stored in a variable logits which is a **n*3 matrix** where n is the number of sentences. _What does each column signify in the output matrix?_

2. For `sentiment_score` variable, we calculate the difference between the fisrt and second column of model's output matrix. _What is the intution behind calculating this difference?_

3. I see that `predictions` variable is always evaluating `None`. _What is the intution behing calculating that variable value?_

**Comment by dhgoratela** [comment]
Let me know if the understanding is correct. for the first question:

1. Every column of model's output matrix is respectively an output class (positive [col1], negative [col2] and neutral [col3])

**Comment by doguaraci** [comment]
> Let me know if the understanding is correct. for the first question:
> 
>     1. Every column of model's output matrix is respectively an output class (positive [col1], negative [col2] and neutral [col3])

That is correct.

2. The sentiment score is one way to interpret the outcome. Subtract negative softmax value from the positive. We use it to transform the score into a continuous range from -1 to 1. 

3. Could you explain a bit more what you mean by this?

**Comment by dhgoratela** [comment]
Sure..
In case I provide only one sentence as input, the `logits `variable evaluates a 1x3 array; Following that we do an argmax which gives a 1x1 array with any one of following value in it (0, 1, 2). Then when we do np.squeeze in this 1x1 array to calculate `predictions` variable, it evaluates nothing..
Pardon me, cannot share a screenshot. Let me know if I could explain you the issue..
